pkg/cli-runtime/printer/table/tabwriter.go

```diff
@@ -34,6 +34,6 @@ const (
 )
 
 // GetNewTabWriter returns a tabwriter that translates tabbed columns in input into properly aligned text.
-func GetNewTabWriter(output io.Writer) *tabwriter.Writer {
-	return tabwriter.NewWriter(output, tabwriterMinWidth, tabwriterWidth, tabwriterPadding, tabwriterPadChar, tabwriterFlags)
+func GetNewTabWriter(output io.Writer, paddingStart bool) *tabwriter.Writer {
```

Comment:
Is there any case where we are setting this to false?

Reply:
If not, you can keep the func signature `GetNewTabWriter(output io.Writer)` and set the `paddingStart` variable to `false` in func `tabwriter.NewWriter` struct

Reply:
Yes, we're actually using it in `False` for `workload list` and `csc list`

Reply:
We're using it in `False` for `csc list` and `wld list` outputs